### PR TITLE
STY: concat strings that should not be seperated

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [ ] closes #xxxx
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/_libs/window/aggregations.pyx [hunk 1]
```diff
@@ -1491,8 +1491,7 @@ cdef ndarray[float64_t] _roll_weighted_sum_mean(float64_t[:] values,
         tot_wgt = np.zeros(in_n, dtype=np.float64)
 
     if minp > win_n:
-        raise ValueError(f"min_periods (minp) must be <= "
-                         f"window (win)")
+        raise ValueError(f"min_periods {minp} must be <= window {win_n}")
```
Comment (MomIsBestFriend) [inline pandas/_libs/window/aggregations.pyx]: The ```f``` did nothing before. should I revert and just delete the ```f```? 
or leave it as it is?

File: pandas/_libs/tslibs/timestamps.pyx [hunk 2]
```diff
@@ -324,8 +323,10 @@ class Timestamp(_Timestamp):
 
         Function is not implemented. Use pd.to_datetime().
         """
-        raise NotImplementedError("Timestamp.strptime() is not implemented."
-                                  "Use to_datetime() to parse date strings.")
+        raise NotImplementedError(
+            "Timestamp.strptime() is not implemented."
```
Comment (simonjayhawkins) [inline pandas/_libs/tslibs/timestamps.pyx]:
```suggestion
            "Timestamp.strptime() is not implemented. "
```

File: pandas/_libs/window/aggregations.pyx (hunk 1 above)
Comment (simonjayhawkins) [reply pandas/_libs/window/aggregations.pyx]: I _think_ leave the message unchanged unless obvious improvement and then should add a brief note in whatsnew that message has been changed.

**Review by simonjayhawkins — COMMENTED** [review]
Thanks @MomIsBestFriend ltgm however probably best in general to leave cython files untouched and wait for black to support formatting cython.

**Comment by MomIsBestFriend** [comment]
>  best in general to leave cython files untouched and wait for black to support formatting cython.

@simonjayhawkins should I close this PR then?

File: pandas/_libs/window/aggregations.pyx (hunk 1 above)
Comment (MomIsBestFriend) [reply pandas/_libs/window/aggregations.pyx]: I think I'll just delete this ```f``` because it didn't do anything any.

**Comment by simonjayhawkins** [comment]
> > best in general to leave cython files untouched and wait for black to support formatting cython.
> 
> @simonjayhawkins should I close this PR then?

I think OK to merge this once the changed message https://github.com/pandas-dev/pandas/pull/30942#discussion_r365592016 is reverted

**Review by WillAyd — APPROVED** [review]
lgtm

**Review by simonjayhawkins — APPROVED** [review]
Thanks @MomIsBestFriend lgtm. ping on green.

**Comment by MomIsBestFriend** [comment]
ping @simonjayhawkins

**Comment by simonjayhawkins** [comment]
Thanks @MomIsBestFriend

**Pull request merged**